patroni/config.py

```diff
@@ -299,7 +302,7 @@ def _build_effective_configuration(self, dynamic_configuration, local_configurat
             config['name'] = pg_config['name']
 
         pg_config.update({p: config[p] for p in ('name', 'scope', 'retry_timeout',
-                          'synchronous_mode', 'maximum_lag_on_failover') if p in config})
+                          'synchronous_mode', 'synchronous_mode_strict', 'maximum_lag_on_failover') if p in config})
```

Comment:
`synchronous_mode_strict` is not used in the `Postgresql` class, therefore we don't need to propagate it.